mailpoet/lib/Util/Url.php

```diff
@@ -69,4 +69,8 @@ public function redirectWithReferer($url = null) {
     }
     exit();
   }
+
+  public function isUsingHttps(string $url): bool {
+    return wp_parse_url($url, PHP_URL_SCHEME) === 'https';
```

Comment:
We use WP functions through are wrapper, as you can see [here](https://github.com/mailpoet/mailpoet/blob/trunk/mailpoet/lib/AdminPages/PageRenderer.php#L119).